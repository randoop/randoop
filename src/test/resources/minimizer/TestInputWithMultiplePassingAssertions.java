import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInputWithMultiplePassingAssertions {

  // This test input file has a passing assertion value. The minimized test suite
  // will use the value found in the assertion as a part of a replacement statement.
  @Test
  public void test1() throws Throwable {
    int i = 7;

    // Passing assertions.
    org.junit.Assert.assertTrue(i == i);
    org.junit.Assert.assertTrue(i == 7);
    org.junit.Assert.assertTrue(i == i != (i != i));
    org.junit.Assert.assertTrue(i == i == (i == i));
    org.junit.Assert.assertTrue(7 == 7);
    org.junit.Assert.assertTrue(i < 8);
    org.junit.Assert.assertTrue(i != 8);
    org.junit.Assert.assertTrue(i * 8 > 1);

    // Failing assertion.
    org.junit.Assert.assertTrue(i == 8);
  }
}
