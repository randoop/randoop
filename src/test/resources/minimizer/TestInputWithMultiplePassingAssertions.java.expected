import org.junit.FixMethodOrder;
import org.junit.Test;
import org.junit.runners.MethodSorters;

@FixMethodOrder(MethodSorters.NAME_ASCENDING)
public class TestInputWithMultiplePassingAssertionsMinimized {

    // This test input file has a passing assertion value. The minimized test suite
    // will use the value found in the assertion as a part of a replacement statement.
    @Test
    public void test1() throws Throwable {
        int i = 0;
        // Failing assertion.
        org.junit.Assert.assertTrue(i == 8);
    }
}
